.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 env:
   - PYTHON_VERSION=2.7
 before_install:
+  - redis-server --version
```

Comment:
It's actually useful since I did need to know what version of the redis server was running. It doesn't harm anything and is good for debugging, so probably best to keep it.